tool/tctl/common/resource_command.go

```diff
@@ -556,7 +555,7 @@ func (rc *ResourceCommand) createClusterMaintenanceConfig(ctx context.Context, c
 		return trace.Wrap(err)
 	}
 
-	fmt.Printf("maintenance window '%s' has been %s\n", cmc.GetName(), UpsertVerb(exists, rc.force))
+	fmt.Printf("maintenance window %q has been %s\n", cmc.GetName(), UpsertVerb(exists, rc.force))
```

Comment:
Similar to UI config, there aren't multiple of these.

Reply:
3cffd6fb0160b7e1c313a308ba98992a38ea8e82